ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift

```diff
@@ -309,17 +309,31 @@ private struct AIChatFeedbackLeoPremiumAdView: View {
 
 enum AIChatFeedbackOption: String, CaseIterable, Identifiable {
   case notHelpful
-  case notWorking
+  case incorrect
+  case unsafeHarmful
   case other
 
-  var id: RawValue { rawValue }
+  var id: RawValue {
+    switch self {
+    case .notHelpful:
+      return "not-helpful"
+    case .incorrect:
+      return "incorrect"
+    case .unsafeHarmful:
+      return "unsafe-harmful"
+    case .other:
+      return "other"
+    }
+  }
```

Comment:
This change doesn't seem needed

Reply:
It is. I need to send exactly just the `id` to the server. The server is seeing localized strings when the user selects an option. But I want the server to see just these strings that aren't localized.

Reply:
Added the comment. Will address the struct vs. enum in the follow up PR I'm opening.